diff -burN libstatgrab-0.10.2.orig/src/statgrab/Makefile.am libstatgrab-0.10.2/src/statgrab/Makefile.am
--- libstatgrab-0.10.2.orig/src/statgrab/Makefile.am	2004-08-01 22:02:25.154366224 +0200
+++ libstatgrab-0.10.2/src/statgrab/Makefile.am	2004-08-01 22:03:01.323867624 +0200
@@ -16,15 +16,3 @@
 LDADD = $(top_builddir)/src/libstatgrab/libstatgrab.la @LINKFLAGS@
 
 install-exec-local:
-if STATGRAB
-if SETGIDBINS
-# Some operating systems need the binary to be setgid kmem
-	chgrp kmem $(DESTDIR)@bindir@/statgrab
-	chmod g+s $(DESTDIR)@bindir@/statgrab
-endif
-if SETUIDBINS
-# Some operating systems need the binary to be setuid root
-	chown root $(DESTDIR)@bindir@/statgrab
-	chmod u+s $(DESTDIR)@bindir@/statgrab
-endif
-endif
