diff -burN libstatgrab-0.9.orig/src/statgrab/Makefile.am libstatgrab-0.9/src/statgrab/Makefile.am
--- libstatgrab-0.9.orig/src/statgrab/Makefile.am	2004-03-13 16:52:00.045466232 +0100
+++ libstatgrab-0.9/src/statgrab/Makefile.am	2004-03-13 17:06:32.595818416 +0100
@@ -16,15 +16,3 @@
 LDADD = $(top_builddir)/src/libstatgrab/libstatgrab.la @LINKFLAGS@
 
 install-exec-local:
-if STATGRAB
-if SETGIDBINS
-# Some operating systems need the binary to be setgid kmem
-	chgrp kmem @bindir@/statgrab
-	chmod g+s @bindir@/statgrab
-endif
-if SETUIDBINS
-# Some operating systems need the binary to be setuid root
-	chown root @bindir@/statgrab
-	chmod u+s @bindir@/statgrab
-endif
-endif
